### ISSUE TITLE
OrdinalCrossEntropy does not use `reduction` argument

### ISSUE THREAD

**Issue by gmgeorg** [ck37/coral-ordinal#12]
OrdinalCrossEntropy inherits from baseline `Loss` class with ` reduction` argument.  However, the reduction argument is then ignored in the actual loss computation of the helper function: `ordinal_loss()` (it uses `reduce_mean()` even if `reduction` != `tf.keras.losses.Reduction.SUM_OVER_BATCH_SIZE` or `AUTO`)
